expyfun/visual/_visual.py

```diff
@@ -826,6 +826,76 @@ def __init__(self, ec, colors=('w', 'k')):
         self.set_radius(1, 1, units='pix')
 
 
+class ProgressBar(object):
+    """A progress bar that can be displayed between sections
+
+    This uses two rectangles, one outline, and one solid to show how much
+    progress has been made in the experiment.
+
+    Parameters
+    ----------
+    ec : instance of ExperimentController
+        Parent EC.
+    pos : array-like
+        4-element array-like with X, Y center and width, height.
+    units : str
+        Units to use. These will apply to all spatial aspects of the drawing.
+        Must be either ''norm'' or ''pix''.
+    colors : list or tuple of matplotlib Colors
+        Colors to fill and outline the bar respectively. Defaults to green and
+        white.
+    text : bool
+        Whether to show the percent done as text below the bar. If ''True'',
+        leave room on the screen below the bar
+
+    Returns
+    -------
+    bar : instance of ProgressBar
+        The progress bar.
+    """
+    def __init__(self, ec, pos, units='norm', colors=('g', 'w')):
+        self._ec = ec
+        if len(colors) != 2:
+            raise ValueError('colors must have length 2')
+        if units != 'norm' and units != 'pix':
```

Comment:
@larsoner WDYT about current implementation vs `if units not in ['norm', 'pix']` ?  I definitely tend toward the latter, but it might just be personal style.

Reply:
I also prefer the latter